src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -0,0 +1,44 @@
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
+/*****************************************************************************
+* OpenRCT2, an open source clone of Roller Coaster Tycoon 2.
+*
+* OpenRCT2 is the work of many authors, a full list can be found in contributors.md
+* For more information, visit https://github.com/OpenRCT2/OpenRCT2
+*
+* OpenRCT2 is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* A full copy of the GNU General Public License can be found in licence.txt
+*****************************************************************************/
+#pragma endregion
+
+#ifdef __linux__
+
+#include <pwd.h>
+#include "../core/Path.hpp"
+#include "Platform2.h"
+
+namespace Platform
+{
+    std::string GetFolderPath(SPECIAL_FOLDER folder)
+    {
+        switch (folder)
+        {
+        case SPECIAL_FOLDER::USER_CACHE:
+        case SPECIAL_FOLDER::USER_CONFIG:
+        case SPECIAL_FOLDER::USER_DATA:
+            {
+                auto home = GetFolderPath(SPECIAL_FOLDER::USER_HOME);
```

Comment:
Why did you remove the use of environment variables?! This is meant as fallback.

Reply:
I will make another PR for that. I just left it to a no-behaviour-change PR.

Reply:
This _does_ change behaviour. Before the PR, it would first check the environment variable, and only then fall back to this.

This very neatly illustrates why I wanted to review it in the first place.